frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -74,32 +83,41 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
     (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
   );
 
+  const rows = combinedDisks.map(
+    (combinedDisk ) => {
+      const size = combinedDisk.getSize();
+      return [
+        combinedDisk.getName(),
+        combinedDisk.getDiskInterface(),
+        size && `${size?.value} ${size?.unit}`,
```

Comment:
`combinedDisk.getReadableSize()` is better here

Reply:
Much better. Thanks for pointing that out. I missed it somehow.